src/styles/style.scss

```diff
@@ -1,3 +1,5 @@
+@import '~bootstrap/dist/css/bootstrap.css';
+
 body {
```

Comment:
the main scss file (style.scss in this case) should only contain `@import` rules. This is probably my fault.